addons/barcodes/models/barcode_rule.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import re
+
+from odoo import api, fields, models, _
+
+
+
+class BarcodeRule(models.Model):
```

Comment:
what is the need of three new line :question: You follow ***PEP8*** standard then as per standard need two new line :smiley: 
![use your brain](https://cloud.githubusercontent.com/assets/14940179/11251343/d55b789e-8e56-11e5-9075-8fc02b950c48.jpg)